instance/models/openedx_instance.py

```diff
@@ -227,7 +227,10 @@ def appserver_is_shut_down(appserver):
             """
             if appserver.status == AppServerStatus.Terminated:
                 return True
-            configuration_failed = appserver.status == AppServerStatus.ConfigurationFailed
+            configuration_failed = (
+                appserver.status == AppServerStatus.ConfigurationFailed or
+                appserver.status == AppServerStatus.Error
+            )
             vm_terminated = appserver.server.status == ServerStatus.Terminated
             return configuration_failed and vm_terminated
```

Comment:
> If the server itself has been terminated, then does it matter what the appserver.status was/is?
@pomegranited Hmm, I suspect it doesn't, but I'm not 100% sure. We still need to check if the config is failed/errored for running VMs, but maybe it should be written to always return `True` if either the appserver or VM statuses are set to "Terminated", and do the more specific checks on the AppServerStatus otherwise. I feel like I'm more confused about the difference between `AppServerStatus.Terminated` and `ServerStatus.Terminated` than anything here.

@itsjeyd since you wrote this function originally, could you chime in?